rfc-097-verify-specific-start-pages.md

```diff
@@ -0,0 +1,136 @@
+# Providing alternative GOV.UK service start pages for the GOV.UK Verify single IDP journey
+
+## Summary
+
+This document describes an approach for providing alternative service start pages 
+that will be displayed to users on a GOV.UK Verify single IDP journey.
+
+## Problem
+
+When following a single IDP journey, the user needs to be redirected via the RP (service) in order
+to pick up a valid SAML request. Initial thoughts were to direct the RP headless start page, 
```

Comment:
I wonder if you could expand on this a little (maybe with some more specific examples spelled out); It strikes me that there's quite a lot implied in this sentence. I'm not really familiar with user journeys or the meanings of the specific vocabulary in Verify (e.g. what is an Identity Provider or Relaying Party?) and so I think I'm missing some important context about the actual need/problem we are trying to address with this RFC. 

It would also be great if you could pop the full version of those acronyms into the text as well please.

Reply:
I've push a revised version with additional text to help explain this. I'm to come over and show a demo if it makes it clearer.

Reply:
👍 thanks that seems clearer now